build-system/compile/compile.js

```diff
@@ -42,9 +41,8 @@ const {writeSourcemaps} = require('./helpers');
 const queue = [];
 let inProgress = 0;
 
-const MAX_PARALLEL_CLOSURE_INVOCATIONS = isCiBuild()
-  ? 10
-  : parseInt(argv.closure_concurrency, 10) || cpus().length;
+const MAX_PARALLEL_CLOSURE_INVOCATIONS =
+  parseInt(argv.closure_concurrency, 10) || cpus().length;
```

Comment:
Does `cpus().length` take into account CPU features such as hyperthreading? It might be possible to increase that limit otherwise

Reply:
Interesting idea. I'll kick off on a separate PR to log the parallelism so we can see how this works on different platform and how we can improve it.